eng/scripts/Language-Settings.ps1

```diff
@@ -495,15 +504,21 @@ function Find-java-Artifacts-For-Apireview($artifactDir, $pkgName)
     return $null
   }
 
-  # Find all source jar files in given artifact directory
-  # Filter for package in "com.azure*" groupId.
-  $artifactPath = Join-Path $artifactDir "com.azure*" $pkgName
-  Write-Host "Checking for source jar in artifact path $($artifactPath)"
-  $files = @(Get-ChildItem -Recurse "${artifactPath}" | Where-Object -FilterScript {$_.Name.EndsWith("sources.jar")})
-  # And filter for packages in "io.clientcore*" groupId.
-  # (Is there a way to pass more information here to know the explicit groupId?)
-  $artifactPath = Join-Path $artifactDir "io.clientcore*" $pkgName
-  $files += @(Get-ChildItem -Recurse "${artifactPath}" | Where-Object -FilterScript {$_.Name.EndsWith("sources.jar")})
+  if ($packageInfo) {
+    $artifactPath = Join-Path $artifactDir $packageInfo.Group $pkgName
+    $files = @(Get-ChildItem "${artifactPath}" | Where-Object -FilterScript {$_.Name.EndsWith("sources.jar")})
+  } else {
+    # Find all source jar files in given artifact directory
```

Comment:
Can we try to eliminate this entire code branch? Given above we are returning null if we don't have a packageInfo? I assume we have already elimianted all cases where we don't have a package info.